src/m365/spp/commands/model/model-apply.spec.ts

```diff
@@ -0,0 +1,541 @@
+    assert.strictEqual(actual.success, true);
+  });
+
+  it('passes validation when required parameters are valid with model id and list URL', async () => {
+    const actual = commandOptionsSchema.safeParse({ webUrl: 'https://contoso.sharepoint.com/sites/sales', contentCenterUrl: 'https://contoso.sharepoint.com/sites/contentCenter', title: 'ModelTitle', listUrl: '/Shared Documents' });
+    assert.strictEqual(actual.success, true);
+  });
+
+  it('passes validation when required parameters are valid with model title and list id and correct viewOption is provided', async () => {
+    const actual = commandOptionsSchema.safeParse({ webUrl: 'https://contoso.sharepoint.com/sites/sales', contentCenterUrl: 'https://contoso.sharepoint.com/sites/contentCenter', title: 'ModelTitle', listId: '421b1e42-794b-4c71-93ac-5ed92488b67d', viewOption: 'NewViewAsDefault' });
+    assert.strictEqual(actual.success, true);
+  });
+
+  it('fails validation when webUrl is not valid', async () => {
+    const actual = commandOptionsSchema.safeParse({ webUrl: 'invalidUrl', contentCenterUrl: 'https://contoso.sharepoint.com/sites/contentCenter', title: 'ModelTitle', listId: '421b1e42-794b-4c71-93ac-5ed92488b67d', viewOption: 'NewViewAsDefault' });
+    assert.strictEqual(actual.success, false);
+  });
+
+  it('fails validation when contentCenterUrl is not valid', async () => {
+    const actual = commandOptionsSchema.safeParse({ webUrl: 'https://contoso.sharepoint.com/sites/sales', contentCenterUrl: 'invalidUrl', title: 'ModelTitle', listId: '421b1e42-794b-4c71-93ac-5ed92488b67d', viewOption: 'NewViewAsDefault' });
+    assert.strictEqual(actual.success, false);
+  });
+
+  it('fails validation when model id is not valid', async () => {
+    const actual = commandOptionsSchema.safeParse({ webUrl: 'https://contoso.sharepoint.com/sites/sales', contentCenterUrl: 'https://contoso.sharepoint.com/sites/contentCenter', id: 'invalidId', listId: '421b1e42-794b-4c71-93ac-5ed92488b67d', viewOption: 'NewViewAsDefault' });
+    assert.strictEqual(actual.success, false);
+  });
+
+  it('fails validation when list id is not valid', async () => {
+    const actual = commandOptionsSchema.safeParse({ webUrl: 'https://contoso.sharepoint.com/sites/sales', contentCenterUrl: 'https://contoso.sharepoint.com/sites/contentCenter', title: 'ModelTitle', listId: 'invalidId', viewOption: 'NewViewAsDefault' });
+    assert.strictEqual(actual.success, false);
+  });
+
+  it('fails validation when viewOption is not valid', async () => {
+    const actual = commandOptionsSchema.safeParse({ webUrl: 'https://contoso.sharepoint.com/sites/sales', contentCenterUrl: 'https://contoso.sharepoint.com/sites/contentCenter', title: 'ModelTitle', listId: '421b1e42-794b-4c71-93ac-5ed92488b67d', viewOption: 'InvalidViewOption' });
+    assert.strictEqual(actual.success, false);
+  });
+
+  it('apply model to document library by id and list id', async () => {
```

Comment:
The test name should be read as a sentence.

`it apply model to document library by id and list id`. So in this case, it should be `applies a` instead of `apply`.
Let's update the other test titles as well.